README.md

```diff
@@ -455,6 +503,14 @@ http://path-to-stream/live/channel-z.ts
   - `catchup-source`: Can contain the full catchup URL (complete with format specifiers) if in `default` mode. Or if the mode is `append` just the query string with format specifiers which will be appended to the channel URL. If omitted the `Query format string` from settings will be appended.
   - `catchup-days`: The number of days in the past catchup is available for.
   - `catchup-correction`: A correction to the time used for catchup stream URL generation. Useful for catchup streams which are geo mis-matched to the wrong time.
+  - `provider`: The name that idenitifes this provider and will be seen in the provider manager. Also that name that can be used for any provider mappings if required. No other provider are valid if this tag is omitted.
+  - `provider-type`: A type which must match one of addon, satellite, cable, aerial, iptv or unknown.
```

Comment:
Isn't it better to highlight valid types with list of code syntax: A type which must match one of `addon`, `satellite`, `cable`, `aerial`, `iptv` or `unknown`